content/references/proptypes.md

```diff
@@ -78,6 +80,45 @@ export const defaultConfig = {
 
 Merchant edits to color fields are stored in RGBA format.
 
+### date
+
+![Date Picker](proptype-date.png)
+
+```javascript
+date: {
+  label: 'Date',
+  type: ElementPropTypes.date
+}
+
+export const defaultConfig = {
+  date: new Date('March 16, 2020')
```

Comment:
@jsvenson don't we want to update this to:
```suggestion
  date: new Date()
```

Reply:
My thought was to have another usage example, but it does make sense to keep them the same. I've made the change.